test/system/application_platform_test.rb

```diff
@@ -69,8 +69,7 @@ def within_current_memberships_table
         refute page.has_content?("Test Platform Application")
       end
       # The tombstones partial won't be rendered if there aren't any tombstoned memberships.
-      # TODO: This eats up a lot of time, look for a better way to check for this.
-      refute page.has_css?("tbody[data-model='Membership'][data-scope='tombstones']")
+      refute all("h2").map { |tag| tag.text }.include?("Former Team Members")
```

Comment:
I wonder if we could get this with just:

```suggestion
      assert_no_text "Former Team Members"
```

Reply:
Or maybe this?

```suggestion
      assert_no_selector "h2", text: "Former Team Members"
```